nightfall-client/src/routes/commitment.mjs

```diff
@@ -90,6 +91,21 @@ router.get('/commitments', async (req, res, next) => {
   }
 });
 
+/**
+ * @description the endpoint that will save a list of commitments
+ */
+router.post('/saveAll', async (req, res, next) => {
+  logger.debug('commitment/saveAll endpoint received POST');
```

Comment:
If we had a standard API, POST `/commitments` would already signal insertion.

I assume the above is not possible, therefore perhaps a "one-word" route should suffice, e.g. `/save`. This could also be `/insert` or `/create`.

Reply:
Changed to save